packages/twenty-server/src/workspace/workspace-sync-metadata/comparators/workspace-relation.comparator.ts

```diff
@@ -51,6 +52,9 @@ export class WorkspaceRelationComparator {
     );
 
     for (const difference of relationMetadataDifference) {
+      const fieldName = difference.path[0];
+      const property = difference.path[difference.path.length - 1];
```

Comment:
I rather scope `fieldName` and `property` in the `CHANGE` type and then add an `if` to check if it's included.
Also now that we have 3 conditions, it's cleaner to use a `switch...case` in my opinion